### PR TITLE
Feature/add rubocop

### PR DESCRIPTION
**Pull request by qr8r**
Install rubocop for pedantic ruby review

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - 'bin/**/*'
+    - 'test/**/*'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+    - 'vendor/**/*'
+    - 'Guardfile'
+
+Rails:
+    Enabled: true
```
Comment (graygilmore) [inline .rubocop.yml]: Would this double indent pass Rubocop!? 😉

Comment (qr8r) [reply .rubocop.yml]: Hahahah

**Comment by qr8r** [comment]
Jenkins test this please

**Comment by qr8r** [comment]
Jenkins test this please

**Comment by qr8r** [comment]
Jenkins test this please

**Comment by jacquesporveau** [comment]
Rubocop style errors are fixed however Brendan is going to be doing some further work with integrating Jenkins thus the CI is still failing.

**Comment by qr8r** [comment]
Jenkins test this please

**Comment by qr8r** [comment]
Jenkins test this please

File: app/controllers/people_controller.rb [hunk 2]
```diff
@@ -47,6 +47,9 @@ def destroy
   private
 
   def person_params
-    params.require(:person).permit(:first_name, :last_name, :email, :password, :password_confirmation, :date_of_birth, :gender)
+    params.require(:person).permit(
+      :first_name, :last_name, :email, :password,
+      :password_confirmation, :date_of_birth, :gender
```
Comment (gregdaynes) [inline app/controllers/people_controller.rb]: Any reason for this formatting over
:first_name,
:last_name,
...

Comment (jacquesporveau) [reply app/controllers/people_controller.rb]: I prefer this style personally but I'll have a look at the style guide and make the appropriate changes.

Comment (qr8r) [reply app/controllers/people_controller.rb]: Haha I actually agree with this :P I didnt actually review the Rubocop corrections :(

Comment (qr8r) [reply app/controllers/people_controller.rb]: The advantage of listing one arg per line is that if a single param changes in the future the git diff will be much easier to read. 

if there are multiple things on one line, and one item in the list changes, git highlights the whole line making it harder to identify what exactly on that line changed.

Ultimately, this is the kind of thing that we should rely on rubocop for. So long as we agree on which style we prefer, we can set up rubocop to enforce it

**Review by gregdaynes — APPROVED** [review]

**Pull request merged**